runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldTableRow.java

```diff
@@ -397,25 +406,35 @@ private JPanel buildWorldField()
 		return column;
 	}
 
-	private static ImageIcon getFlag(WorldRegion region)
+	private static ImageIcon getFlag(World world, Set<Integer> westCoastWorlds)
 	{
-		if (region == null)

```

Comment:
you probably should not remove the region null check since the code https://github.com/runelite/api.runelite.net/blob/master/http-api/src/main/java/net/runelite/http/api/worlds/WorldRegion.java#L71 returns null if the region is unknown. Which might not happen now, but could if they added a new region unexpectedly. This would break the panel due to throwing out of populate().